understanding/20/headings-and-labels.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Headings and Labels</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>A page's content is described in headings and labels</dd>
-		<dt>Author task</dt><dd>Provide descriptive headings and labels</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>A page's content is described in headings and labels.</dd>
+         <dt>What to do</dt><dd>Provide descriptive headings and labels.</dd>
+         <dt>Why it's important</dt><dd>Users with reduced sight or cognition can orient themselves.</dd> 
```

Comment:
@rainbreaw commented
> 'Reduced cognition' implies that there is equivalence between cognitive disabilities and intellect. Suggest rewording to
```suggestion
         <dt>Why it's important</dt><dd>Users with reduced sight or cognitive disabilities can orient themselves.</dd> 
```

Reply:
That was from earlier language suggested to us when we were trying to find generic descriptors for broad senses/abilities, such as "reduced vision" (including blind), "reduced hearing" (including deaf), etd. The idea was that 'reduced cognition' was equivalent to "Reduce cognitive function". When we lose this broad terms, it gets messier since we have to add more words...

At any rate, I'll adopt your suggestion, but I'm reversing it so no one reads this as 'reduced cognitive disabilities'. This also seems like a good time to acknowledge this helps everyone...
```suggestion
         <dt>Why it's important</dt><dd>People, especially those with cognitive disabilities or reduced sight, can orient themselves.</dd> 
```